### DISCUSSION TITLE
Target #49

### DISCUSSION THREAD

**Discussion by Anneh82** [cssbattle/forum#166] [Q&A] [answered]
Hi,

I’m new here and also new in CSS (just started three weeks ago). I think the challenges are really cool to practice! So I did a battle before, it wasn’t perfect but still got 100% match (i did the slide thing and i still saw a difference. Now I did target #49 (stay at home) and its perfect when I do the slide but I only got a 95.5% match. What do I do wrong here? I want to learn :). I’m still a noob in CSS. Still so many things to learn. But i really think its fun.

I did this:

`div class="square1"` `/div` `div class="square2"` `/div` `divclass="rectangle"` `/div` `divclass="triangle"` `/div`

`style` body { background: #6592CF; display: flex; justify-content: center; align-items: center; position: relative; } div.square1 { background: #243D83; width: 150px; height: 130px; border-radius: 10px; position: absolute; margin: 35px 0px 0px 0px; } div.square2 { background: #EEB850; width: 50px; height: 50px; border-radius: 10px 10px 0px 0px; position: absolute; margin: 75px 0px; } div.rectangle { background: #EEB850; width: 100px; height: 10px; border-radius: 10px; position: absolute; z-index: 1; } div.triangle { width: 0px; height: 0px border-top: 100px solid transparent; border-right: 100px solid transparent; border-bottom: 100px solid #243D83; border-left: 100px solid transparent; margin: -100px 0px 0px 0px; position: relative; } `/style`

Thank you so much :)

**Comment by rafehimself** [comment]
You forgot about  `<div> `s in your code

**Reply by Anneh82** [reply]
Oh sorry I didn't put that into my question. I did do that in my coding tho. Like this: <div class="square1" </div . But if I put in the closing brackets here, it will not show Up in my answer.

**Reply by rafehimself** [reply]
I see, use `code` brackets to insert your code to the post
![image](https://user-images.githubusercontent.com/58576132/140922967-70c4fa5c-da8c-412a-a21e-05a0d47ed12a.png)

**Reply by Anneh82** [reply]
Thanks!

**Comment by rafehimself** [comment, marked as answer]
```
<div class="square1"></div>
<div class="square2"></div>
<div class="rectangle"></div>
<div class="triangle"> </div>

<style> body { background: #6592CF; display: flex; justify-content: center; align-items: center; position: relative; }
 div.square1 { background: #243D83; width: 150px; height: 130px; border-radius: 10px; position: absolute; top:112 }
  div.square2 { background: #EEB850; width: 50px; height: 50px; border-radius: 10px 10px 0px 0px; position: absolute; bottom:42 }
  div.rectangle { background: #EEB850; width: 100px; height: 10px; border-radius: 10px; position: absolute; z-index: 1; }
  div.triangle { width: 0px; height: 0px border-top: 100px solid transparent; border-right: 100px solid transparent; border-bottom: 100px solid #243D83; border-left: 100px solid transparent; margin: -100px 0px 0px 0px; position: relative; }

</style> 
```

**Reply by Anneh82** [reply]
Thank you so much!!!
